mappings/net/minecraft/network/chat/MessageSignature.mapping

```diff
@@ -1,3 +1,26 @@
 CLASS net/minecraft/unmapped/C_shphkjbi net/minecraft/network/chat/MessageSignature
+	FIELD f_bpkdlana BYTES I
```

Comment:
Having `BYTES` and `bytes` seems unnecessarily confusing. 
Is `BYTES` the size of the `bytes` array? If so maybe:
```suggestion
	FIELD f_bpkdlana NUM_BYTES I
```